lms/templates/courseware/progress.html

```diff
@@ -59,15 +59,25 @@ <h1 class="progress-certificates-title">${_("Course Progress for Student '{usern
                     <% post_url = reverse('generate_user_cert', args=[unicode(course.id)]) %>
                     <div class="msg-content">
                         <h2 class="title">${_("Your certificate is available")}</h2>
+                        %if show_cert_web_view:
+                        <p class="copy">${_("You can now view your certificate.")}</p>
+                        %else:
```

Comment:
I did not see any text in JIRA ticket. I have added this on my own. we can update or change it as per suggestions.